src/cpp/session/modules/SessionConsole.R

```diff
@@ -14,3 +14,46 @@
 #
 #
 
+.rs.addJsonRpcHandler("console_follow_hyperlink", function(url, text, params)
+{
+  # key1=value1:key2:value2 pattern, see https://iterm2.com/documentation-escape-codes.html
+  if (identical(params, "")) {
+    params <- list()
+  } else {
+    params <- strsplit(params, ":")[[1L]]
+
+    names  <- sapply(params, function(p) sub("=.+$", "", p))
+    params <- lapply(params, function(p) sub("^[^=]+=", "", p))
+    names(params) <- names
+  }
+  
+  if (identical(url, "rstudio:help")) {
+    .rs.showHelpTopic(params$topic, params$package) 
+  } else if (identical(url, "rstudio:vignette")) {
+    print(vignette(params$topic, package = params$package))
+  } else if (grepl("^rstudio:viewer:", url)) {
+    url <- sub("^rstudio:viewer:", "", url)
+    return(.rs.api.viewer(url))
+  } else if (grepl("^file://", url)) {
```

Comment:
I'm not sure this is correct if `url` is relative to the current working directory ? the project ? 

Maybe this should be another `rstudio:` prefixed scheme ?

Reply:
Yeah, the working directory might change after the printing, and maybe there is no project. Maybe in general we should encourage absolute paths, and cli would also convert relative paths to absolute ones.

Reply:
Good point. Let's worry about this on the `cli` side.